aii-ks/src/main/pan/quattor/aii/ks/schema.pan

```diff
@@ -57,7 +57,7 @@ type structure_ks_logging = {
 type structure_ks_ks_info = {
     "ackurl"	: type_absoluteURI
     "acklist"	? type_absoluteURI[]
-    "auth"		: string[] = list ("enableshadow", "enablemd5")
+    "auth"		: string[] = list ("enableshadow", "passalgo=sha512")
```

Comment:
@jrha are you proposing to but a regex on the string to match either the begin of sha512 or md5? (have no clue if it s feasible)